examples/elasticsearch-incremental/elasticsearch-incremental-blueprint.yaml

```diff
@@ -0,0 +1,439 @@
+          curl  -k -u "elastic:$PASSWORD"  -X PUT "${ES_URL}/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+          {
+            "persistent": {
+              "xpack.monitoring.collection.enabled": false
+            }
+          }
+          '
+          curl  -k -u "elastic:$PASSWORD"  -X POST "${ES_URL}/_watcher/_stop?pretty"
+          
+          echo "delete all datastreams and indices" 
+          curl  -k -u "elastic:$PASSWORD" -X PUT "${ES_URL}/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+          {
+            "persistent": {
+              "action.destructive_requires_name": false
+            }
+          }
+          '
+          curl  -k -u "elastic:$PASSWORD" -X DELETE "${ES_URL}/_data_stream/*?expand_wildcards=all&pretty"
+          curl  -k -u "elastic:$PASSWORD" -X DELETE "${ES_URL}/*?expand_wildcards=all&pretty"
+
+          echo "restore all"
+          # TODO manage the monitoring of the restoration process          
+          curl  -k -u "elastic:$PASSWORD" -X POST "${ES_URL}/_snapshot/k10_repo/$SNAPSHOT_NAME/_restore?pretty" -H 'Content-Type: application/json' -d'
+          {
+            "indices": "*",
+            "include_global_state": true
+          }
+          '
+
+          echo "Restart service geoip, machine learning, monitoring and watcher"
+          curl  -k -u "elastic:$PASSWORD"  -X PUT "${ES_URL}/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+          {
+            "persistent": {
+              "ingest.geoip.downloader.enabled": true
+            }
+          }
+          '
+          curl  -k -u "elastic:$PASSWORD"  -X POST "${ES_URL}/_ilm/start?pretty"
+          curl  -k -u "elastic:$PASSWORD"  -X POST "${ES_URL}/_ml/set_upgrade_mode?enabled=false&pretty"
+          curl  -k -u "elastic:$PASSWORD"  -X PUT "${ES_URL}/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+          {
+            "persistent": {
+              "xpack.monitoring.collection.enabled": true
+            }
+          }
+          '
+          curl  -k -u "elastic:$PASSWORD"  -X POST "${ES_URL}/_watcher/_start?pretty"
+
+          echo "Reenable destructive_requires_name"
+          curl  -k -u "elastic:$PASSWORD" -X PUT "${ES_URL}/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+          {
+            "persistent": {
+              "action.destructive_requires_name": null
+            }
+          }
+          '
+  # limitation : deletion need a cluster with the same name in the same namespace be up and running
+  # if you don't have that then just create it and proceed the deletions of the restorepoint
+  delete:
+    inputArtifactNames: 
+    # use it with  for instance {{ .ArtifactsIn.s3Snap.KeyValue.repoPath }} {{ .ArtifactsIn.s3Snap.KeyValue.snapshotName }} 
+    #                           {{ .ArtifactsIn.s3Snap.KeyValue.esClusterName }} {{ .ArtifactsIn.s3Snap.KeyValue.esClusterNamespace  }}
```

Comment:
do we need these comment? if yes, can we please improve them.
`use it with for instance` is not very intuitive.

Reply:
done